backend/src/main/resources/graphql/main.graphqls

```diff
@@ -295,12 +295,13 @@ type TestResult {
   symptomOnset: LocalDate
   genderOfSexualPartners: [String]
   deviceType: DeviceType
-  results: [MultiplexResult]
+  results: [MultiplexResult!]
```

Comment:
⚠️ This is so that the `TestResult` type plays well with [`getResultForDisease`](https://github.com/CDCgov/prime-simplereport/blob/00710d95f297e65a30bdf3b869e1915e1f906ba0/frontend/src/app/utils/testResults.tsx#L51) without having to break too much further down with how `MultiplexResults` aka `MultiplexResult[]` is defined. I think this is safe to say that even if we already allow `results` to be null or undefined, if we have a defined, non-null list for `results`, the members of that list should not be null or undefined. If there are any null safety concerns here I can opt for the other route of updating `MultiplexResults` to handle a list with nullable entries